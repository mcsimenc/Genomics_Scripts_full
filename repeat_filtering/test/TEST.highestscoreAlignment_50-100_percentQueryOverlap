gnl|Theca1.0|Tc00_g045221	scf7180000009622 (dbseq-nr 2060) [64256,65446]	55.41561712846348
gnl|Theca1.0|Tc00_g039291	scf7180000007625 (dbseq-nr 63) [67984,70680]	63.18131256952169
gnl|Glyma1.01|PACid:16278609	scf7180000007696 (dbseq-nr 134) [612802,614220]	85.62367864693447
gnl|Frave2.0|gene01269	scf7180000007957 (dbseq-nr 395) [7849,10018]	53.50230414746544
gnl|Phoda3.0|PDK_30s701911g001	scf7180000007908 (dbseq-nr 346) [207104,209033]	52.53886010362694
gnl|Ambtr1.0.27|AmTr_v1.0_scaffold00024.244	scf7180000009819 (dbseq-nr 2257) [409349,411351]	72.94058911632551
gnl|Aquco1.0|PACid:18150727	scf7180000009992 (dbseq-nr 2430) [191273,192675]	51.31860299358517
gnl|Glyma1.01|PACid:16264313	scf7180000009808 (dbseq-nr 2246) [87620,90847]	52.88104089219331
gnl|Glyma1.01|PACid:16278609	scf7180000007945 (dbseq-nr 383) [231336,235627]	52.84249767008388
gnl|Glyma1.01|PACid:16264313	scf7180000008998 (dbseq-nr 1436) [388855,392212]	50.83382966051221
gnl|Nelnu1.0|NNU_012536-RA	scf7180000010121 (dbseq-nr 2559) [276913,278495]	80.16424510423246
gnl|Glyma1.01|PACid:16264313	scf7180000009421 (dbseq-nr 1859) [420736,422550]	89.42148760330578
gnl|Theca1.0|Tc07_g016860	scf7180000010013 (dbseq-nr 2451) [437378,438849]	54.61956521739131
gnl|Theca1.0|Tc01_g016582	scf7180000008998 (dbseq-nr 1436) [553135,554480]	73.32838038632985
gnl|Theca1.0|Tc01_g016582	scf7180000010251 (dbseq-nr 2689) [791321,793255]	51.007751937984494
gnl|Theca1.0|Tc00_g039291	scf7180000010419 (dbseq-nr 2857) [1537576,1540297]	78.80235121234386
