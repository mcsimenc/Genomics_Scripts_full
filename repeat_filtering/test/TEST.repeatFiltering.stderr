cat
GetPlantTribesOGinfo.py OrthoMCL summary
GetPlantTribesOGinfo.py OrthoMCL annotation
GetPlantTribesOGinfo.py Orthofinder summary
GetPlantTribesOGinfo.py Orthofinder annotation
GetPlantTribesOGinfo.py GFam summary
GetPlantTribesOGinfo.py GFam annotation
diffLines.py
